README.md

```diff
@@ -27,5 +27,7 @@ Players must enter accurate input as requested.
 ### External dependencies
 TBC
 
-### Usage
-TBC
+### Logging
+When launching the app from the command line:<br>
+Appending '-d' enables the display of debug messages in the console.<br>
+Appending '-f' enables the logging of debug messages to a file named 'ttt.log'.
```

Comment:
Is this still true ?

Reply:
Indeed not. I used to think I was relatively good at detail :). The README.md is now updated to reflect reality.

Reply:
Looking forward to demoing at AOB.